mage_integrations/mage_integrations/destinations/bigquery/templates/config.json

```diff
@@ -2,5 +2,5 @@
   "path_to_credentials_json_file": "path_to_credentials_json_file",
   "project_id": "",
   "dataset": "",
-  "table": ""
```

Comment:
actually, let's keep 'table' here. we still need it when we test destination individually via bash command.

Reply:
@wangxiaoyou1993 you should still be able to add it to the config, it just won't be shown to the user through the UI.